docs/generated_code.md

```diff
@@ -94,8 +95,9 @@ For the following message declaration:
 message Example {}
 ```
 
-we generate a class called `Example`, which extends the base class [Message](https://github.com/bufbuild/protobuf-es/blob/tstamm/add-docs/packages/protobuf/src/message.ts#L40)
-provided by [@bufbuild/protobuf](../packages/protobuf). See the [runtime API documentation](#runtime-api) for details.
+we generate a class called `Example`, which extends the base class [Message](../packages/protobuf/src/message.ts#L40)
```

Comment:
Do you want to use a permalink here to a specific revision? Otherwise the line number may not be accurate if changes are made to this file in the future.

Reply:
There are many other ways docs and code can run out of sync, but yeah, let's use permalinks for those. Done.